README.md

```diff
@@ -169,8 +169,6 @@ docker-compose down --volumes db
 If you have any questions or feedback, do not hesitate to contact us via GitHub or Gitter:
 
 - https://github.com/saleor/saleor/discussions
-- https://gitter.im/mirumee/saleor
```

Comment:
why do we remove that? I thought it's still supported, I saw @karolkielecki comments there no more than month old.

Reply:
Due to SEO, we are moving to Github discussions